dask/bytes/hdfs3.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function, division, absolute_import
+
+import posixpath
+
+from .glob import generic_glob
+from ..base import tokenize
+
+import hdfs3
+
+
+class HDFS3HadoopFileSystem(object):
+    sep = "/"
+
+    def __init__(self, **kwargs):
+        self.fs = hdfs3.HDFileSystem(**kwargs)
+
+    @classmethod
+    def from_hdfs3(cls, fs):
+        out = object.__new__(cls)
+        out.fs = fs
+        return out
```

Comment:
Why do we have this?

Reply:
Testing purposes. Wrapping an existing hdfs client in the dask wrapper, rather than creating one from scratch.